sway/main.c

```diff
@@ -223,6 +223,25 @@ void enable_debug_flag(const char *flag) {
 	}
 }
 
+static sway_log_importance_t convert_wlr_log_importance(
+		enum wlr_log_importance importance) {
+	switch (importance) {
+	case WLR_ERROR:
+		return SWAY_ERROR;
+	case WLR_INFO:
+		return SWAY_INFO;
+	default:
+		return SWAY_DEBUG;
+	}
+}
+
+static void handle_wlr_log(enum wlr_log_importance importance,
+		const char *fmt, va_list args) {
+	static char sway_fmt[1024];
+	snprintf(sway_fmt, sizeof(sway_fmt), "[wlr] %s", fmt);
```

Comment:
Unless there's a contract on the length of `fmt`, it's probably best to check the return value of snprintf.

Reply:
I think a truncated log message isn't a big deal.